packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -289,7 +289,7 @@ function apply_user_configs(configs, hostname)
     os.execute("/usr/bin/lime-config")
     -- Start sharing lime-defaults and reboot
     share_defualts()
-    remove_lock_file()
+    remove_lock()
```

Comment:
this should be fbw.remove_lock() ?

Reply:
Indeed!
So important are the UnitTests, right?